geps/gep-3171/index.md

```diff
@@ -171,4 +175,40 @@ spec:
           numerator: 5
           denominator: 1000
 ```
+
 This would result in 0.5% of requests going to `foo-v1` to be mirrored to `foo-v2`.
+
+## Examples for East/West traffic
+
+Similarly, here is how it would be for mesh.
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: http-filter-mirror
+  labels:
+    service: mirror-service
+spec:
+  parentRefs:
+  - group: ""
+    kind: Service
+    name: mirror-service
+  rules:
+  - backendRefs:
+    - name: foo-v1
+      port: 8080
+    filters:
+    - type: RequestMirror
+      requestMirror:
+        backendRef:
+          name: foo-v2
+          port: 8080
+        percent: 42
+```
+
+## Conformance Details
+
+A new Supported Feature with the following name SupportHTTPRouteRequestPercentageMirror is introduced along with a corresponding conformance test.
+
+<!-- TODO(liorlieberman) Different supportedFeature for mesh> -->
```

Comment:
Reminder not to forget this. Will this have separate conformance tests?

Reply:
I was planning to keep this as a TODO and iterate. WDYT? need to check if any mesh supports this today

Reply:
It's fine if this is a follow up, but if so, should be an issue instead of a TODO

Reply:
added a issue link inside the todo instead of my name.